app/controllers/stories_controller.rb

```diff
@@ -358,22 +360,16 @@ def set_user_json_ld
         "@id": URL.user(@user)
       },
       "url": URL.user(@user),
-      "sameAs": [],
+      "sameAs": user_same_as,
       "image": ProfileImage.new(@user).get(width: 320),
       "name": @user.name,
-      "email": "",
-      "jobTitle": "",
+      "email": @user.email_public && @user.email.presence,
```

Comment:
this will result in `email: false` if the email is not public:

```ruby
[1] pry(main)> email = false && "bob@acme.com"
=> false
[2] pry(main)> email
=> false
```

it should be:

```ruby
"email": @user.email_public ? @user.email : nil
```